docs/userGuide/syntax/frontmatter.md

```diff
@@ -23,6 +23,11 @@
 Should you need more expressive formatting, or encounter any issues when formatting the frontmatter, note that the frontmatter follows the [YAML](https://yaml.org/refcard.html) spec.
 </box>
 
+<box type="warning" seamless>
+
+If a page has multiple front matters, it will take the last front matter by default. You may make use of the [omitFrontmatter]({{ base_url }}/userGuide/reusingContents.html#includes) attribute to omit the front matters that are not needed.
```

Comment:
I think it's fine to include the warning here. You may also want to elaborate on the second part that `omitFrontmatter` is only applicable when using the `<include>` feature.